client/api/analytics/AnalyticsPublisherService.cs

```diff
@@ -17,9 +17,7 @@ internal class AnalyticsPublisherService
         private static readonly string VariationIdentifierAttribute = "variationIdentifier";
         private static readonly string TargetAttribute = "target";
         internal static readonly ConcurrentDictionary<Target, byte> SeenTargets = new();
-        private static readonly ConcurrentDictionary<Target, byte> StagingSeenTargets = new();
         private static readonly string SdkType = "SDK_TYPE";
-        private static readonly string AnonymousTarget = "anonymous";
```

Comment:
@davejohnston I will add the anonymous stuff back in tomorrow, per our chat in Slack

Reply:
private target added back in